process-engine/process_engine_runtime
# ✨ Add EndEvent name and ID to CallActivity result

**Changes:**

1. Include the EndEvent- name & ID with each CallActivity's result token.
2. Cleanup the CallActivity test BPMNs.
3. Cleanup CallActivity integration tests and add assertions for EndEvent- name & ID.

**Issues:**

Closes #280

PR: #283

## How can others test the changes?

- Run a Process with BPMN Studio that makes use of CallActivities
- Go to the `Inspect` view and inspect the ProcessTokens for the ProcessInstance
- See that the onExit token for the CallActivity now contains three properties:
    - result: The CallActivity's final result
    - endEventName: The name of the EndEvent that finished the CallActivity's SubProcess
    - endEventId: The ID of the EndEvent that finished the CallActivity's SubProcess

## PR-Checklist

Please check the boxes in this list after submitting your PR:
